packages/template-retail-react-app/app/components/toggle-card/index.jsx

```diff
@@ -34,6 +34,15 @@ export const ToggleCard = ({
     children,
     ...props
 }) => {
+    let editButtonAriaLabel = 'Edit'
```

Comment:
This string will need to have translation attached to it.

Reply:
Replaced it with formatMessage